### ISSUE TITLE
Issue: Dropdown does not pass A11 Accessibility linting 

### ISSUE THREAD

**Issue by ParkerSchmitt** [hoaphantn7604/react-native-element-dropdown#214]
I have a project that uses this component along with https://github.com/aryella-lacerda/react-native-accessibility-engine to help enforce accessibility within our UI.

Accessibility assertions failing output: 
```
DropDown > View > TouchableWithoutFeedback

    Problem:  "This component is pressable but the user hasn't been informed that it behaves like a button/link/radio"
    Solution: "Set the 'accessibilityRole' prop to button or link or imagebutton or radio or tab"

 View > View > TouchableWithoutFeedback

    Problem:  "This component is pressable but the user hasn't been informed that it behaves like a button/link/radio"
    Solution: "Set the 'accessibilityRole' prop to button or link or imagebutton or radio or tab"

    Problem:  "This button is not accessible (selectable) to the user"
    Solution: "Set the 'accessible' prop to 'true' or remove it (pressables are accessible by default)"
```

**Comment by carlos3g** [comment]
Resolved in the maintained fork [`@carlos3g/element-dropdown`](https://github.com/carlos3g/element-dropdown) v2.13.0+: trigger has the `combobox` role with `expanded` / `disabled` state, items have `button`, chips have `button` + a remove hint, and the modal sets `accessibilityViewIsModal`. Full breakdown in the [accessibility doc](https://carlos3g.github.io/element-dropdown/docs/accessibility). Drop-in.
